### PR TITLE
setStroke() in printLinearGradient() and printRadialGradient()

### PR DESCRIPTION
**Pull request by Wondermarin**
Added stroke param in printLinearGradient() and printRadialGradient().

### CODE REVIEW

**Comment by Wondermarin** [comment]
> You could rename `printLinearGradient` to `printLinearColorGradient` and add `printLinearStrokeGradient`, likewise the radial version. This will fix the problem that all of `...args` are in fact, required arguments, and `stroke = false` is positioned first being an optional one.
> 
> Next version of canvas-constructor is a semver-major, so it makes sense for this breaking change.

Yes, that would be much better.

File: src/lib/canvas.js [hunk 1]
```diff
@@ -1026,9 +1052,28 @@ class Canvas {
 	 * @chainable
 	 * @see https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/createRadialGradient
 	 */
-	printRadialGradient(...args) {
+	printRadialColorGradient(...args) {
 		const gradient = this.createRadialGradient(...args);
 		return this.setColor(gradient);
+  }
+  
+  	/**
```
Comment (kyranet) [inline src/lib/canvas.js:1060]:
```suggestion
	/**
```

File: src/lib/canvas.js [hunk 2]
```diff
@@ -979,16 +979,42 @@ class Canvas {
...
 	 *     ])
 	 *     .addRect(10, 10, 200, 100)
 	 */
-	printLinearGradient(...args) {
+	printLinearColorGradient(...args) {
 		const gradient = this.createLinearGradient(...args);
 		return this.setColor(gradient);
+  }
+
+  	/**
```
Comment (kyranet) [inline src/lib/canvas.js:994]:
```suggestion
	/**
```

File: src/lib/canvas.js [hunk 3]
```diff
@@ -979,16 +979,42 @@ class Canvas {
...
 	 *         { position: 0.25, color: 'red' },
 	 *         { position: 0.5, color: 'blue' }
 	 *     ])
 	 *     .addRect(10, 10, 200, 100)
 	 */
-	printLinearGradient(...args) {
+	printLinearColorGradient(...args) {
 		const gradient = this.createLinearGradient(...args);
 		return this.setColor(gradient);
+  }
```
Comment (kyranet) [inline src/lib/canvas.js:992]:
```suggestion
	}
```

File: src/lib/canvas.js [hunk 4]
```diff
@@ -1026,9 +1052,28 @@ class Canvas {
 	 * @chainable
 	 * @see https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/createRadialGradient
 	 */
-	printRadialGradient(...args) {
+	printRadialColorGradient(...args) {
 		const gradient = this.createRadialGradient(...args);
 		return this.setColor(gradient);
+  }
```
Comment (kyranet) [inline src/lib/canvas.js:1058]:
```suggestion
	}
```

**Pull request merged**